animations/tween-animations.md

```diff
@@ -0,0 +1,727 @@
+---
+layout: page
+title: Tween Animations in Flutter
+permalink: /animations/tween-animations.html
+---
+
+* TOC Placeholder
+{:toc}
+
+The animation system in Flutter is based on typed
+[`Animation`](https://docs.flutter.io/flutter/animation/Animation-class.html)
+objects. Widgets can either incorporate these animations in their build
+functions directly by reading their current value and listening to their
+state changes or they can use the animations as the basis of more elaborate
+animations that they pass along to other widgets.
+
+## Animation<double>
+
+In Flutter, an Animation object knows nothing about what is onscreen.
+An Animation is an abstract class that understands its current value
+and its state (completed or dismissed). One of the more commonly used
+animation types is Animation<double>.
```

Comment:
here and elsewhere (e.g. the header), you need to put backticks around class names otherwise the `<` characters end up interpreted as HTML.